homework012/sort_user.go

```diff
@@ -2,58 +2,121 @@ package homework012
 
 import (
 	"fmt"
+	"log"
+
+	"xorm.io/xorm"
 )
 
 // 問題5
 // User構造体を実装してください。
+type User struct {
+	Id    int    `xorm:"id"`
+	Name  string `xorm:"name"`
+	Class int    `xorm:"class"`
+	Age   int    `xorm:"age"`
+}
 
 // 問題5 を実装してください。
 // 中身を適宜変更してください。引数も変更して大丈夫です。
-func insertUsers() {
+func insertUsers(engine xorm.Engine, users []User) {
+	for _, user := range users {
+		_, err := engine.Table("user").Insert(user)
```

Comment:
```suggestion
		_, err := engine.Table("user").Insert(users)
```
このように配列をそのままInsertする事も出来ます。